scripts/secnetperf.ps1

```diff
@@ -0,0 +1,300 @@
+    exit
+}
+
+$RemoteAddress = $Session.ComputerName
+Write-Output "Successfully conencted to peer: $RemoteAddress"
+
+# Make sure nothing is running from a previous run.
+if ($isWindows) {
+    Invoke-Command -Session $Session -ScriptBlock {
+        Get-Process | Where-Object { $_.Name -eq "secnetperf.exe" } | Stop-Process
+    }
+} else {
+    Invoke-Command -Session $Session -ScriptBlock {
+        Get-Process | Where-Object { $_.Name -eq "secnetperf" } | Stop-Process
+    }
+}
+
+# Copy the artifacts to the peer.
+Write-Output "Copying files to peer..."
+if ($isWindows) {
+    Invoke-Command -Session $Session -ScriptBlock {
+        Remove-Item -Force -Recurse "C:\_work" -ErrorAction Ignore
+    }
+    Copy-Item -ToSession $Session .\artifacts -Destination C:\_work\quic\artifacts -Recurse
+    Copy-Item -ToSession $Session .\scripts -Destination C:\_work\quic\scripts -Recurse
+} else {
+    Invoke-Command -Session $Session -ScriptBlock {
+        Remove-Item -Force -Recurse "/home/secnetperf/_work" -ErrorAction Ignore
+        mkdir /home/secnetperf/_work
+    }
+    Copy-Item -ToSession $Session ./artifacts -Destination /home/secnetperf/_work/artifacts -Recurse
+    Copy-Item -ToSession $Session ./scripts -Destination /home/secnetperf/_work/scripts -Recurse
+}
+
+try {
+
+mkdir .\artifacts\logs | Out-Null
+
+# Prepare the machines for the testing.
+
+if ($isWindows) {
+    Write-Output "Skipping prepare machine for now on Windows..."
+    # Write-Output "Preparing machines for testing..."
+    # .\scripts\prepare-machine.ps1 -ForTest
+
+    # Invoke-Command -Session $Session -ScriptBlock {
+    #     C:\_work\quic\scripts\prepare-machine.ps1 -ForTest
+    # }
+} else {
+    Write-Output "Skipping prepare machine for now on Linux..."
+
+    # Write-Output "Preparing machines for testing..."
+    # .\scripts\prepare-machine.ps1 -ForTest
+
+    # Invoke-Command -Session $Session -ScriptBlock {
+    #     /home/secnetperf/_work/scripts/prepare-machine.ps1 -ForTest
+    # }
+}
+
+# Logging to collect quic traces while running the tests.
+
+if ($LogProfile -ne "" -and $LogProfile -ne "NULL") { # TODO: Linux back slash works?
+    Write-Output "Starting logging with log profile: $LogProfile..."
+    .\scripts\log.ps1 -Start -Profile $LogProfile
+}
+
+# Run secnetperf on the server.
+Write-Output "Starting secnetperf server..."
+
+if ($isWindows) {
+    $Job = Invoke-Command -Session $Session -ScriptBlock {
+        C:\_work\quic\artifacts\bin\windows\x64_Release_schannel\secnetperf.exe -exec:maxtput
+    } -AsJob
+} else {
+    $Job = Invoke-Command -Session $Session -ScriptBlock {
+        $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:/home/secnetperf/_work/artifacts/bin/linux/x64_Release_openssl/"
+        chmod +x /home/secnetperf/_work/artifacts/bin/linux/x64_Release_openssl/secnetperf
+        /home/secnetperf/_work/artifacts/bin/linux/x64_Release_openssl/secnetperf -exec:maxtput
+    } -AsJob
+}
```

Comment:
These paths need to not be hard coded, but instead take the input parameters into account to build up the path.